docs/en/latest/plugins/jwt-auth.md

```diff
@@ -174,29 +207,20 @@ curl http://127.0.0.1:9080/apisix/admin/consumers -H 'X-API-KEY: edd1c9f034335f1
 }'
 ```
 
-This plugin uses rsa public key from consumer configuration and uses the private key directly fetched from vault.
```

Comment:
Please keep this line. It's important to deliver the message.

Reply:
I had moved the line to above the code block. We do mention this.

Reply:
I had moved the line to above the code block. We do mention this.

Reply:
Gotcha!